components/licensor/ee/cmd/sign.go

```diff
@@ -90,6 +92,7 @@ func init() {
 	rootCmd.AddCommand(signCmd)
 
 	signCmd.Flags().String("domain", "", "domain for which the license is valid")
+	signCmd.Flags().String("customer-id", "", "analytics customer ID")
```

Comment:
I'm on the fence about whether it's worthwhile to expose customer-id in the licensor `sign` command. Leaving this value out seems a bit surprising but we're not planning on using this heavily going forward. I'd appreciate an outside opinion - should we keep this internal only or plumb it through to the command?